nuspec/nuget/Cake.Http.nuspec

```diff
@@ -17,8 +17,14 @@
       <tags>cake script build httpclient web</tags>
    </metadata>
    <files>
-      <file src="Cake.Http.dll" target="lib/net45" />
-      <file src="Cake.Http.pdb" target="lib/net45" />
-      <file src="Cake.Http.xml" target="lib/net45" />
+      <file src="net45/Cake.Http.dll" target="lib/net45" />
```

Comment:
According to [parameters.cake](https://github.com/cake-contrib/Cake.Recipe/blob/develop/Cake.Recipe/Content/parameters.cake#L272) the default value for ShouldRunDotNetPack is false, so I think there is no changes to be done.